assets/sass/protocol/base/elements/_forms.scss

```diff
@@ -1,12 +1,12 @@
+/* stylelint-disable declaration-colon-newline-after */
```

Comment:
Does this need to be at the top of the file? It looks like the same feature is disabled for a line below so this might be redundant. If we do need a disable comment at the top it should probably go after the license block.